src/services/offline/MockModels.ts

```diff
@@ -147,3 +150,15 @@ export const OfflineAccountNames = (address: Address) => new AccountNames(addres
 export const OfflineNamespaceNames = (namespaceId: NamespaceId) => new NamespaceName(namespaceId, 'mocknamespace');
```

Comment:
OfflineNamespaceNames why this is using mocknames?

Reply:
There's a request done for some NamespaceIds when selecting a profile/account, so if you don't return an array with the same size of the ids request it breaks.